github.js

```diff
@@ -724,6 +724,24 @@
           }
           _request("GET", url, null, cb);
       };
+
+      // starring
+      this.isStarred = function(owner, repo, cb) {
+        _request("GET", "/user/starred/" + owner + '/' + repo, null, function(err,res) {
```

Comment:
Hi there

No I have no time to work with it. Please use it in any way you like.

Best regards
Serge

On Sun, Jan 24, 2016, 3:45 PM Aurelio De Rosa <notifications@github.com>
wrote:

> In github.js
> <https://github.com/michael/github/pull/206#discussion_r50634816>:
>
> > @@ -724,6 +724,24 @@
> >            }
> >            _request("GET", url, null, cb);
> >        };
> > +
> > +      // starring
> > +      this.isStarred = function(owner, repo, cb) {
> > +        _request("GET", "/user/starred/" + owner + '/' + repo, null, function(err,res) {
>
> You can have cb as the last parameter so you can avoid creating an
> anonymous function that just invoke the callback.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/michael/github/pull/206/files#r50634816>.
>